### ISSUE TITLE
option to disable 'length-zero-no-unit'

### ISSUE THREAD

**Issue by dmerc** [stylelint/stylelint#2027]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

i am extending "stylelint-config-standard", it sets 'length-zero-no-unit' to true. I would like to disable it, but config `"rules": { "length-zero-no-unit": false }` is invalid.

> Which rule, if any, is this issue related to?

length-zero-no-unit

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "length-zero-no-unit": false
  }
}
```

> Which version of stylelint are you using?

7.5.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

using webpack-stylelint-loader

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
Invalid Option: Unexpected option value "false" for rule "length-zero-no-unit"
```

**Comment by m-allanson** [comment]
@dmerc try using `null`, rather than `false`.

https://github.com/stylelint/stylelint/blob/master/docs/user-guide/faq.md#how-do-i-disable-a-rule
